web/Controllers/PropertyController.cs

```diff
@@ -123,4 +124,21 @@ public ActionResult AddNewProperty_Cancel()
         // Go to View Properties page
         return RedirectToAction("ViewProperties", "Landlord");
     }
-}
+
+    [HttpGet]
+    [Route("/property/view")]
+    public ActionResult ViewProperty(int propertyId)
+    {
+        PropertyDbModel? model = _propertyService.GetPropertyByPropertyId(propertyId);
+        if (model == null)
+        {
+            _logger.LogWarning("View property did not return the property as expected");

```

Comment:
Could this message say something like "Property with ID {id} does not exist"

Reply:
ok done